frontend/app/src/components/Sidebar/SidebarNav.test.tsx

```diff
@@ -139,80 +128,273 @@ describe("SidebarNav", () => {
     expect(viewButton).not.toBeInTheDocument()
   })
```

Comment:
Should probably remove this test (starting on line 120) - not sure what the test title even means lol